pages/glossary/adjacent.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Adjacent
+key: adjacent
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+[Bounding boxes][] A and B are _adjacent_ if at least one point from box A is at the distance of 1 point from one point in box B.
+
+The distance between two points is obtained by adding the horizontal distance between the two points with the vertical distance between the two points.
```

Comment:
That doesn't sound right. So something with x:0,y0 has a distance of 2 to x:1,y:1? I don't think we should do that.

Reply:
What don't you think we should do? Consider (0,0) and (1,1) to not be adjacent or that the distance between them should not be 2? Or both?

In my opinion they are not adjacent, but its just an opinion and I don't feel too strong about that.

But I do feel the distance is 2 pixels (I have to move 1 pixel right and 1 pixel down to get from one point to the other). Pixels must be integer, therefore it cannot be the square root of 2.

Reply:
I was trying to come up with a distance metric that would always result in whole pixels. But, you're right, for distance we don't need to require that. I'll delete the definition of distance from this definition.